### ISSUE TITLE
Possible typo in lewm.yaml: max_epochs=100

### ISSUE THREAD

**Issue by NiccoloCavagnero** [lucas-maes/le-wm#52]
Hi, 

first of all, congratulations for the amazing work!

I would like to point out that there might be a typo in the configuration file for training (lewm.yaml):
while the paper reports 10 epochs for training across the 4 datasets, the configuration file now sets max_epochs to 100.

This can be misleading for someone that might have missed this details in the paper.

Best, 
Niccolò

**Comment by lucas-maes** [comment]
hi! it's not a problem haha we did setup training for 100 epoch but stop after 10 epoch you can change to 10 but will likely have to adapt the learning rate!
